ivy/functional/frontends/paddle/nn/functional/activation.py

```diff
@@ -31,3 +31,11 @@ def selu(
 
 tanh = paddle_tanh
 log_softmax = paddle_log_softmax
+
+
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def hardtanh(x, min=-1.0, max=1.0):
```

Comment:
I think that you should mimik how the selu function above write its function signature, specifically how it specifes that some arguments that include name are optional keyword_only arguments.